apps/bfd-server-ng/src/main/java/gov/cms/bfd/server/ng/claim/model/ProviderHistory.java

```diff
@@ -98,10 +98,7 @@ HumanName toFhirName(Optional<String> legacyLastName) {
     return name;
   }
 
-  Optional<DomainResource> toFhirNpiTypePartD(ClaimTypeCode claimTypeCode) {
-    if (!claimTypeCode.isBetween(1, 4)) {
-      return Optional.empty();
-    }
+  Optional<DomainResource> toFhirNpiTypePartD() {
```

Comment:
This isn't specific to part D anymore.
```suggestion
  Optional<DomainResource> toFhirNpiType() {
```